### PR TITLE
Feature/swiftgen

### PR DESCRIPTION
**Pull request by vcoutasso**
## Description
Adds and configures swiftgen to the project

## Why?
Introduces type-safety to access project assets

### CODE REVIEW

File: LearningJourney/Modules/Core/UI/UI/Resources/Assets.xcassets/Colors/CardBackground.colorset/Contents.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xF7",
-          "green" : "0xF1",
-          "red" : "0xF1"
+          "blue" : "0.969",
```
Comment (pastre) [inline LearningJourney/Modules/Core/UI/UI/Resources/Assets.xcassets/Colors/CardBackground.colorset/Contents.json]: Did you mean to change those values?

Comment (vcoutasso) [reply LearningJourney/Modules/Core/UI/UI/Resources/Assets.xcassets/Colors/CardBackground.colorset/Contents.json]: Actually I didn't, must have been some kind of mistake

Comment (pastre) [reply LearningJourney/Modules/Core/UI/UI/Resources/Assets.xcassets/Colors/CardBackground.colorset/Contents.json]: maybe thats whats breaking ci

**Review by pastre — APPROVED** [review]
LGTM. Created #39 . PRs like this one shouldn't fail on CodeCov

**Pull request merged**